webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <ds-text tag="h5" color="soft" class="spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </ds-text>
```

Comment:
let's refactor out all styleguide components? you can use a `h5` tag here and apply `CSS` to make it soft, etc...
and if you give this a class of `title`, for example, you can target it with `CSS` like: 
```
.follow-list {
  > .title {
   ....
  }
}
```
which says only target the class title that is a direct child of `follow-list`

Reply:
Was just hesitating to do that. So basically, whenever I come across a ds-* tag, refactor it out?
The question is then, what design-tokens do I use? Just any that fit and if nothing applies, create one? We don't want to end up with 20 definitions of a card-heading is what I'm concerned about. @alina-beck 

Anyway, there is a ds-input later on, I'll start with by turning it into a base component.

Reply:
yes, please. There should be design tokens for all the sizes that are used in `ds-text`... otherwise maybe see if one of them looks good, ie not too big or too small.
As far as the `ds-input`, I would probably leave that because it would actually involve migrating the `ds-input` component and refactoring, ie separate ticket(code creep).
My larger point was that if it's possible to accomplish the same with `html` tags and `CSS`, let's do that, so we don't make it harder on ourselves to refactor later.

Reply:
done in 72dfeb3